packages/wrangler/templates/startDevWorker/ProxyWorker.ts

```diff
@@ -37,6 +37,7 @@ export class ProxyWorker implements DurableObject {
 
 	proxyData?: ProxyData;
 	requestQueue = new Map<Request, DeferredPromise<Response>>();
+	requestRetryQueue = new Map<Request, DeferredPromise<Response>>();
```

Comment:
Is there any need for these to be Maps rather than just arrays?
I don't see anywhere that we look up the promise based on the request.

Reply:
The original idea was to ensure a single entry per request but now there are 2 queues that doesn't imply the guarantee it did before. But it also makes deletes easier.